src/HelpOverlayTrigger.js

```diff
@@ -15,21 +17,27 @@ const HelpOverlayTrigger = () => {
           href={ 'https://autoedit.gitbook.io/autoedit-3-user-manual/' }
           target={ '_blank' }
           rel="noopener noreferrer"
+          onClick={ () => trackEvent({ category: 'global - help dropdown', action: 'click', name: 'view user manual' }) }
         >
           User manual.
         </a>
         <br />
         This is a <strong>slightly different version</strong> of the Digital
         Paper Edit, but the general workflows should be the same.
         Should you require login details, or more assistance,{' '}
-        <a href="mailto:dpe@bbcnewslabs.co.uk">get in touch with Newslabs</a>!
+        <a
```

Comment:
Alli's updated this code recently with the new user manual so could you pull in her changes from master?